src/client/components/HeroBanner.less

```diff
@@ -0,0 +1,114 @@
+@import (reference) "../styles/custom.less";
+@import (reference) "../styles/modules/variables.less";
+
+.HeroBanner {
+  position: relative;
+  height: 240px;
+  border-radius: 5px;
+  margin-bottom: 16px;
+  background: url("/images/banner.png");
+
+  @media @small-max-width {
+    margin: 0 8px 16px 8px;
+  }
+
+  &__close {
+    position: absolute;
+    top: 8px;
+    right: 16px;
+    color: @white;
+    z-index: 1;
+    cursor: pointer;
+
+    &:link,
+    &:visited,
+    &:hover,
+    &:focus,
+    &:active {
+      text-decoration: none;
+    }
+
+    &:hover {
+      color: @white-gainsboro;
+    }
+  }
+
+  &__primary >  {
+    border-radius: 5px;
+    font-weight: bolder;
+    font-size: 14px;
+    padding: 8px 12px;
+    background-color: @green;
+    transition: background-color 0.5s ease-in-out;
+
+    &:link,
+    &:visited,
+    &:focus,
+    &:active {
+      color: @white;
+      text-decoration: none;
+    }
+
+    &:hover {
+      color: @white;
+      text-decoration: none;
+      background-color: darken(@green, 10%);
+    }
+  }
+
+  &__secondary >  {
+    border-radius: 5px;
+    font-weight: bolder;
+    font-size: 14px;
+    padding: 6px 10px;
+    border: 2px solid @white;
+    transition: all 0.5s ease-in-out;
+
+    &:link,
+    &:visited,
+    &:focus,
+    &:active {
+      color: @white;
+      text-decoration: none;
+    }
+
+    &:hover {
+      color: @blue-linkWater;
+      text-decoration: none;
+      border-color: @blue-linkWater;
+    }
+  }
+
+  .container {
+    height: 100%;
+  }
+
+  &__container {
+    display: flex;
+    height: 100%;
+    padding: 16px 0;
+
+    @media @small-max-width {
+      padding: 16px;
+    }
+
+    &__content {
+      margin: auto 0;
```

Comment:
I think you might be able to get rid of this if you are going to make the herobanner take up the entire screen width. And underneath `&__content` somewhere after around `line 87` you can add the following flex properties: 
```
align-items: center;
justify-contenter: center;
```
And then doing a few changes around like so:

in `HeroBanner.js`:
![screen shot 2017-11-29 at 8 03 16 pm](https://user-images.githubusercontent.com/5067716/33407635-464d4ad2-d540-11e7-9865-37b042d0459d.png)

in `Page.js`:
![screen shot 2017-11-29 at 8 02 38 pm](https://user-images.githubusercontent.com/5067716/33407644-541ef372-d540-11e7-830d-02cbf567d984.png)

in `HeroBanner.less`:
![screen shot 2017-11-29 at 8 02 53 pm](https://user-images.githubusercontent.com/5067716/33407661-64c873a6-d540-11e7-9ddb-84501bbd221c.png)

Here's what I have in my local, after checking out your fork and branch, and testing it out.

![screen shot 2017-11-29 at 7 56 55 pm](https://user-images.githubusercontent.com/5067716/33407431-4c1e2e64-d53f-11e7-948c-499ea285e627.png)

sry for the screenshots, but the git GUI that I use doesn't let me copy and paste lol.....